engine/src/main/java/org/hibernate/validator/constraintvalidation/HibernateConstraintValidatorContext.java

```diff
@@ -68,4 +68,6 @@
 	 * @since 5.2
 	 */
 	TimeProvider getTimeProvider();
+
+	HibernateConstraintValidatorContext withInfo(Object info);
```

Comment:
I find _withInfo_ a bit too generic. WDYT about _withViolationContext_ or something like that?